agents/TabularQ.py

```diff
@@ -7,6 +7,7 @@ def __init__(self, state_shape, num_acts, params):
         self.alpha = params['alpha']
         self.gamma = params['gamma']
         self.epsilon = params['epsilon']
+        self.nu = .95
```

Comment:
For now it would probably be best to leave this at 1. Using this sort of scaling would be a novel contribution in this paper and may detract from our primary novel contribution of the bayesian uncertainty for exploration.

We'll want to save this in our back pocket in case the bayesian uncertainty fails to work without it.